traveler-ios-ui/TravelerKitUI/ViewControllers/PartnerOfferingsFetchViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  PartnerOfferingsFetchViewController.swift
+//  TravelerKitUI
+//
+//  Created by Omar Padierna on 2019-12-01.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+public protocol PartnerOfferingsFetchViewControllerDelegate: class {
+    func partnerOfferingsFetchViewController(_ controller: PartnerOfferingsFetchViewController, didFinishWith purchaseForm: PurchaseForm)
+}
+
+open class PartnerOfferingsFetchViewController: UIViewController {
+
+    var product: PartnerOfferingItem?
+    weak var delegate: PartnerOfferingsFetchViewControllerDelegate?
+
+    private var offerings: [PartnerOfferingGroup]?
+
+    override open func viewDidLoad() {
+        super.viewDidLoad()
+        guard let product = product else {
+            Log("PartnerOfferingsItem is nil", data: nil, level: .warning)
+            return
+        }
+
+        Traveler.fetchPartnerOfferings(product: product, delegate: self)
+
+    }
+
+    override open func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        switch (segue.identifier, segue.destination) {
+        case (_, let vc as OfferingsSelectViewController):
+            vc.offeringsGroup = offerings
+            vc.product = product
+            vc.delegate = self
+        case (_, let vc as ErrorViewController):
+            vc.delegate = self
+            vc.retryButtonString = "Dismiss"
```

Comment:
Add a TODO in the ErrorViewController to rename retryButton and retryButtonString to actionButton and actionButtonString to keep it more generic.

Reply:
Renamed them instead.